spec/aws_loader_spec.rb

```diff
@@ -1,25 +1,53 @@
+# Dummy class for testing module
+class ModuleTest
+  include AwsLoader
+end
+
 RSpec.describe AwsLoader do
 
   let (:mod_test) { ModuleTest.new }
 
   context 'aws.yml file' do    
     context '#configure_aws_file' do
-      it 'should return false for aws.yml file check and call #configure_aws_file and create a new .yml file' do
+      it 'should call #configure_aws_file' do
         stub_const('AwsLoader::AWS_PATH', nil)
-        allow(File).to receive(:exists?).and_return(false)
-        expect(STDOUT).to receive(:puts).and_return('No `aws.yml` file present!')
-        expect(STDOUT).to receive(:puts).and_return('Created file and set the appropriate values!')
+        allow(File).to receive(:exists?).with(AwsLoader::AWS_PATH).and_return(false)
         expect(mod_test).to receive(:generate_aws_yml_file)
         mod_test.configure_aws_file
       end
-      
-      it 'should return true for aws.yml file check and check for aws config values in the file' do
-        stub_const('AwsLoader::AWS_PATH', File.join(QIt.root, 'aws.yml'))
+    end
+
+    # context 'no aws key values' do
+    #   it 'should log error for no values for the aws config keys in it' do
+    #     stub_const('AwsLoader::AWS_PATH', File.join(Application.root, 'aws.yml'))
+    #     allow(File).to receive(:exists?).with(AwsLoader::AWS_PATH).and_return(true)
+    #     allow(File).to receive(:read).with(AwsLoader::AWS_PATH).and_return({'aws' => {"access_key_id"=>"", "secret_access_key"=>"", "region"=>""}})
+    #     allow(YAML).to receive(:load).and_return({ 'aws' => { "access_key_id"=>"", "secret_access_key"=>"", "region"=>"" } })
+    #     expect(Application).to receive_message_chain(:log, :error).with("Fill in the appropriate values for the aws.yml file")
+    #     mod_test.configure_aws_file
```

Comment:
Replace this line with `expect{ mod_test.configure_aws_file }.to raise_error(SystemExit)`.
Uncomment this test and run.